metals/src/main/scala/scala/meta/internal/metals/codeactions/CodeActionCommandData.scala

```diff
@@ -5,15 +5,17 @@ import org.eclipse.lsp4j.Location
 import org.eclipse.lsp4j.TextDocumentPositionParams
 
 trait CodeActionCommandData {
-  val uri: String
+  def uri: String
   val actionType: String
```

Comment:
Maybe let's change this to `def` too
```suggestion
  def actionType: String
```

Reply:
Changed